KP5-Topic2/KP1-Topic2/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst

```diff
@@ -0,0 +1 @@
+C:\Users\Luciano\Documents\Topic-2\KP5-Topic2\KP1-Topic2\src\main\java\com\aguirre\KP1_Topic2\App.java
```

Comment:
Please include the folder target in .gitignore file to avoid to upload to the repo these generated files